lua/nvim-tree.lua

```diff
@@ -363,15 +363,15 @@ local function setup_autocommands(opts)
 
   create_nvim_tree_autocmd("BufReadPost", {
     callback = function()
-      if filters.config.filter_no_buffer or renderer.config.highlight_opened_files then
+      if filters.config.filter_no_buffer or renderer.config.highlight_opened_files ~= "none" then
         reloaders.reload_explorer()
```

Comment:
```suggestion
        utils.debounce("BufReadPost:filter_buffer", 10, function()
          reloaders.reload_explorer()
        end)
```
Actually please change them to something like this. This would make reload only happen once if a bunch of BufReadPost happened in quick succession.

Reply:
I like this.

`update_focused_file.debounce_delay` default `15` has recently been added, to cope with telescope's extra buffer switching.

Please use that value for now, even thought it is only for focused. We should do a full audit of such instances but Not Today.

Reply:
Raised #1836